### ISSUE TITLE
Importing `.mjs` in Chrome 72 throws syntax error

### ISSUE THREAD

**Issue by ajmeyghani** [MikeMcl/bignumber.js#222]
I'm getting the following syntax error when trying to load the module from the `.mjs` file using a script tag with type module in Chrome 72:

```
export BigNumber = clone(); // Uncaught SyntaxError: Unexpected token export
```

My setup is very simple:

**main.js**

```
import BigNumber from "./node_modules/bignumber.js/bignumber.mjs";
console.log(BigNumber("25"));
```

**html**

```
<script type="module" src="./main.js"></script>
```

But then when I change the export line to the following, it works fine:

```
// EXPORT
BigNumber = clone();
export {
  BigNumber,
  BigNumber as default,
};
```

Is the syntax used not supported by the latest version of Chrome? I'm very curious...

**Comment by MikeMcl** [comment]
Yes, I mistakenly removed the `var` before I published the new version yesterday.

It should be

```js
export var BigNumber = clone();
```

I will publish the correction later today.

Thanks very much for the report.

**Comment by MikeMcl** [comment]
Fixed in *v8.1.1*.
